src/test/shell/bazel/cpp_darwin_integration_test.sh

```diff
@@ -80,5 +80,19 @@ EOF
       fail "//cpp/rpaths:test execution failed, expected to return 0, but got $?"
 }
 
+function test_osx_binary_strip() {
+  mkdir -p cpp/osx_binary_strip
+  cat >cpp/osx_binary_strip/BUILD <<EOF
+cc_binary(
+  name = "main",
+  srcs = ["main.cc"],
+)
+EOF
+  cat >cpp/osx_binary_strip/main.cc <<EOF
+int main() { return 0; }
+EOF
+  assert_build //cpp/osx_binary_strip:main.stripped >& $TEST_log || fail "//cpp/osx_binary_strip:main.stripped didn't build"
```

Comment:
Can you pls move the fail onto the next line.

Reply:
This matches the existing style. Note that I think there might be something wrong with the current test code -- https://github.com/bazelbuild/bazel/issues/4085 -- would you like me to update both as part of this change?

Reply:
Does it? Line 80 in this file is the desired formatting. Let me investigate #4085 first, we run this test on the CI and I don't think we have this issue there. I'll take a look.

Reply:
Hmm you know what, just fix it pls, the fail shouldn't be there twice even if it worked. I didn't know assert_build also fails.

Reply:
Done.